Submitted By: Jeff L
Date: 2004-03-26
Initial Package Version: 0.18
Origin: blfs.support
Description: Fixes compilation of rep-gtk with gtk-2.4 installed.

$LastChangedBy$
$Date$

--- rep-gtk-0.18/rep-gtk.h.orig	2004-03-26 14:10:18.000000000 -0500
+++ rep-gtk-0.18/rep-gtk.h	2004-03-26 14:11:15.000000000 -0500
@@ -291,10 +291,10 @@
 
 GtkWidget*
 gtk_radio_menu_item_new_with_label_from_widget (GtkRadioMenuItem *group,
-						gchar            *label);
+					  const gchar            *label);
 GtkWidget*
 gtk_radio_menu_item_new_with_mnemonic_from_widget (GtkRadioMenuItem *group,
-						   gchar            *label);
+					  const gchar               *label);
 GtkWidget* gtk_radio_menu_item_new_from_widget (GtkRadioMenuItem *group);
 GtkWidget* gtk_pixmap_new_interp (char *file, GtkWidget *intended_parent);

--- rep-gtk-0.18/gtk-compat.c.orig	2004-03-26 14:57:08.000000000 -0500
+++ rep-gtk-0.18/gtk-compat.c	2004-03-26 14:57:43.000000000 -0500
@@ -64,7 +64,7 @@
 
 GtkWidget*
 gtk_radio_menu_item_new_with_label_from_widget (GtkRadioMenuItem *group,
-						gchar            *label)
+					const gchar            *label)
 {
   GSList *g = group? gtk_radio_menu_item_group (group) : NULL;
   return gtk_radio_menu_item_new_with_label (g, label);
@@ -72,7 +72,7 @@
 
 GtkWidget*
 gtk_radio_menu_item_new_with_mnemonic_from_widget (GtkRadioMenuItem *group,
-						   gchar            *label)
+					const gchar            *label)
 {
   GSList *g = group? gtk_radio_menu_item_group (group) : NULL;
   return gtk_radio_menu_item_new_with_mnemonic (g, label);
